ext/afform/core/Civi/Api4/Action/CustomGroup/GetAfforms.php

```diff
@@ -285,13 +286,16 @@ private function generateTabForm($item): array {
       'permission' => ['access all custom data'],
       'title' => $item['title'],
       'icon' => $item['icon'],
+      'summary_weight' => 100 + ($item['weight'] ?? 0),
     ];
-    if ($item['extends'] === 'Contact') {
+    if (CoreUtil::isContact($item['extends'])) {
       $afform['placement'] = ['contact_summary_tab'];
-    }
-    elseif (CoreUtil::isContact($item['extends'])) {
-      $afform['placement'] = ['contact_summary_tab'];
-      $afform['summary_contact_type'] = [$item['extends']];
+      if (!empty($item['extends_entity_column_value'])) {
+        $afform['summary_contact_type'] = (array) $item['extends_entity_column_value'];
+      }
+      elseif ($item['extends'] !== 'Contact') {
```

Comment:
Just looking again, should this be "else"? Are contact_sub_type keys unique across all contact types? (I hope so, but just a thought).

I.e. Could you have `VIP` Individual and a `VIP` household as different contact sub types?

Reply:
No, there's a unique index on `name`
https://github.com/civicrm/civicrm-core/blob/7befa582c4d1ad545616ef2990349656bd441e42/schema/Contact/ContactType.entityType.php#L20-L25